src/renderer/components/Transaction/TransactionForm/TransactionFormTransfer.vue

```diff
@@ -427,9 +416,7 @@ export default {
           return this.walletNetwork.apiVersion === 1 // Return true if it's v1, since it has a static fee
         }
       },
-      vendorField: {
-        maxLength: maxLength(64)
-      },
+      vendorFielf: {},
```

Comment:
`vendorField` or, since your are not using it anymore, remove it.

Reply:
Actually, if I don't include validation for vendorField, I see a lot of errors due to that in console

Reply:
Yes, that's is because you are using the Vuelidate field (`$v.form.vendorField`). Since it's not performing any validation, you could have replaced it with `form.vendorField` and removed the `vendorField` validation.